OpenRA.Mods.Common/Widgets/ProductionPaletteWidget.cs

```diff
@@ -375,6 +379,28 @@ public override bool HandleKeyPress(KeyInput e)
 			return toBuild != null ? HandleEvent(toBuild, MouseButton.Left, batchModifiers) : false;
 		}
 
+		bool SelectProductionBuilding()
+		{
+			var viewport = worldRenderer.Viewport;
+			var selection = World.Selection;
+
+			if (CurrentQueue == null)
+				return true;
+
+			var facility = CurrentQueue.MostLikelyProducer().Actor;
+
+			if (facility == null)
+				return true;
```

Comment:
Had to put this null check here because this was crashing mods with classic production when using the hotkey at the start of the game (before the MCV is unpacked).